omnibus/Gemfile.lock

```diff
@@ -287,8 +289,8 @@ GEM
     multipart-post (2.2.3)
     net-scp (4.0.0)
       net-ssh (>= 2.6.5, < 8.0.0)
-    net-sftp (4.0.0)
-      net-ssh (>= 5.0.0, < 8.0.0)
```

Comment:
NOTE: This bump for `net-sftp` was done to fix an issue https://github.com/chef/chef/pull/13284/files but it is getting reverted by dependabot